packages/delayedInbox-l2msg/scripts/normalTx.js

```diff
@@ -1,31 +1,28 @@
 const { providers, Wallet } = require('ethers')
 const hre = require('hardhat')
 const ethers = require('ethers')
-const { hexDataLength } = require('@ethersproject/bytes')
-const { L1ToL2MessageGasEstimator} = require('@arbitrum/sdk/dist/lib/message/L1ToL2MessageGasEstimator')
 const { arbLog, requireEnvVariables } = require('arb-shared-dependencies')
-const { L1TransactionReceipt, L1ToL2MessageStatus } = require('@arbitrum/sdk')
 requireEnvVariables(['DEVNET_PRIVKEY', 'L2RPC', 'L1RPC', 'INBOX_ADDR'])
```

Comment:
We could use SDK to load the Inbox address instead gf having users to set it as an env var

Reply:
Yea, but currently the sdk doesn't store the address information for goerli network, so it still let users to add in.

Reply:
Right, we can change this while SDK 2.0.10 (migration) is released.